imports/plugins/core/orders/client/containers/orderSummaryContainer.js

```diff
@@ -116,25 +116,17 @@ class OrderSummaryContainer extends Component {
 }
 
 const composer = (props, onData) => {
-  const userId = Meteor.userId();
-  const userSub = Meteor.subscribe("UserProfile", userId);
   const orderSub = Meteor.subscribe("Orders");
-  let profile = {};
-
-  if (userSub.ready() && orderSub.ready()) {
-    if (typeof userId === "string") {
-      const userProfile = Accounts.findOne(userId);
-      if (!userProfile) {
-        return profile;
-      }
-      profile = userProfile.profile.addressBook[0];
-    }
 
+  if (orderSub.ready()) {
+    // Find current order
     const order = Orders.findOne({
       "_id": props.orderId,
       "shipping._id": props.fulfillment._id
     });
 
+    const profile = order.shipping[0].address;
```

Comment:
Since this is an address not a profile I wonder if we shouldn't call this variable `profileAddress`? I realize that's how it was in the original code but maybe we can make this easier to read.